### PR TITLE
improved version of collapsable left menu

### PR DESCRIPTION
**Pull request by lcarcone**
I improved the previous version of collapsable left menu and fixed the menu at the top of the window.

### CODE REVIEW

File: html/qtypes-site-results.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
 <link rel='alternate' type='application/atom+xml' href='/2002/09/wbs/myQuestionnaires.atom' title='ATOM feed of my Questionnaires' />
 <script src='/2002/09/wbs/autolinker-dom.js'></script><script src='/2002/09/wbs/addLinks.js'></script>
 <script src='../js/menu.js'></script>
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
```
Comment (tripu) [inline html/qtypes-site-results.html]: Better to get jQuery from the same location we're importing it from on the other pages: `https://www.w3.org/scripts/jquery/2/jquery.min.js`

**Review by tripu — CHANGES_REQUESTED** [review]
* Since you're *replacing* both `menu.png` and `close.png`, why not simply overwrite both files, without changing their names? That way: a) we don't keep the old, unused versions around; and b) you don't have to bother editing the source code to change the names there.
* We should display the icons only when JS is enabled; otherwise they're pointless and will only confuse users.
* There's an odd white triangle in a corner of the element where the icons are situated.

File: css/site-wbs-style.css [hunk 2]
```diff
@@ -787,12 +787,14 @@ div.w3c_leftCol h3.category span.ribbon {
   border: 0;
 } 
 
-#leftCol_menu a:hover {
-  background-color: inherit;
-} 
-
 #leftCol_menu {
-  position: fixed; left: 10px; top: 115px;
+    width: 45px;
```
Comment (tripu) [inline css/site-wbs-style.css]: This not being `100%` is what causes the little white triangle I mentioned.

**Review by tripu — CHANGES_REQUESTED** [review]
image

**Comment by tripu** [comment]
@lcarcone, I can work on some of these comments tomorrow (eg, hiding the icons by default and showing them iif the UA's running JS; tweaking the appearance to get rid of the white triangle).

**Comment by lcarcone** [comment]
I overwritten menu.png. For close.png,  I hat to cut it a little and as it is already used elsewhere in the pages I prefer to keep a different icon.
I have added some <noscript> tags to hide the button when JS is disabled, it should be ok I guess.
I have also updated the location of jquery.min.js
Concerning the white triangle at the bottom right of the element, it is defined in the class 'category' (advanced.css:112) and the background image https://www.w3.org/2008/site/images/category-bg-fold.png. Let's merge the PR as it for the moment and raise an issue, I will address it before to commit in wbs code

Thanks

**Comment by lcarcone** [comment]
Tripu,
I used a modified local version of the background image https://www.w3.org/2008/site/images/category-bg-fold.png (I removed the white triangle).
What do you think ?

Thanks

**Review by tripu — APPROVED** [review]

**Comment by tripu** [comment]
@lcarcone, I reverted some of your changes, and committed what I think are better ways to achieve those goals.
Please review and, if :+1:, merge yourself!

**Comment by lcarcone** [comment]
Hi Tripu,
Thanks for the improvements (and sorry for the delay).
I looked at gitlab and I liked the way it manages the sidebar (a bottom fixed ribbon), I made some changes to have a similar behavior. I think it is nicer than to have a button at the top of the menu (and we don't have to manage the scroll of the left menu).
What do you think ?

**Review by tripu — CHANGES_REQUESTED** [review]
@lcarcone, the thing is: with your last version, the left-side menu does not stay stuck when there's vertical scroll&hellip; I think that is useful, because the user has access to that menu at all times. I still prefer our previous version.

**Review by tripu — APPROVED** [review]

**Comment by tripu** [comment]
## Changes introduced

### Before, top

before-top

### After, top

after-top

### Before, scrolled

before-scrolled

### After, scrolled

after-scrolled

### Before, collapsed

before-collapsed

### After, collapsed

after-collapsed

In sumary:

* Icon moved to bottom, instead of left dangling there near (but not *at*) the top
* Descriptive label added
* Most important (although not shown in the *before/after* screenshots): the icon does not obscure (nor is obscured) but the left-hand-side menu when one starts scrolling down (that is a bug as it is now).

**Comment by tripu** [comment]
@koalie, do we have MarComm's blessing for this change? My last comment summarises what it looks like. It isn't a big change, but makes the collapsible menu a bit more usable imho, and fixes a bug.

**Comment by koalie** [comment]
@tripu Yes. Thanks!
